decentraland/unity-renderer
# Fix: Improving memory management when loading scenes

## What does this PR change?

This PR enforces some objects' references to null to limit memory garbage accumulated while doing multiple teleporting from one scene to another.

...

## How to test the changes?

0. Go to https://play.decentraland.zone/?renderer-branch=fix/scene-loading-leak
1. Try some sequential teleport from Genesis Plaza to other locations and back, waiting a bit between teleports
2. Check for crashes if the platform is stable even between multiple teleports
3. hopefully, enjoy a bit less filled RAM 

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
